source/guides/applications/logging.md

```diff
@@ -4,13 +4,13 @@ title: Guides - Logging
 
 # Logging
 
-Within an [Container project](/guides/architectures/container), for each application, there is a Ruby constant that gives us access to the logger.
+Within a [Container project](/guides/architectures/container), each application has a Ruby constant that gives us access to its logger.
```

Comment:
I changed 'the logger' to 'its logger'. @jodosha is it true that each application in a container has its own Logger? Or is it just many references to the same logger? Kind of small distinction, but I want to make sure we're telling people the truth.